backends/platform/sdl/win32/win32.cpp

```diff
@@ -145,6 +145,46 @@ bool OSystem_Win32::openUrl(const Common::String &url) {
 	return true;
 }
 
+Common::String OSystem_Win32::getScreenshotsPath() {
+	char screenshotsPath[MAXPATHLEN];
+
+	OSVERSIONINFO win32OsVersion;
+	ZeroMemory(&win32OsVersion, sizeof(OSVERSIONINFO));
+	win32OsVersion.dwOSVersionInfoSize = sizeof(OSVERSIONINFO);
+	GetVersionEx(&win32OsVersion);
+	// Check for non-9X version of Windows.
+	if (win32OsVersion.dwPlatformId != VER_PLATFORM_WIN32_WINDOWS) {
+		// Use the Application Data directory of the user profile.
+		if (win32OsVersion.dwMajorVersion >= 5) {
+			if (!GetEnvironmentVariable("APPDATA", screenshotsPath, MAXPATHLEN))
+				error("Unable to access application data directory");
+		}
+		else {
```

Comment:
Formatting: } else {

Please refer to: http://wiki.scummvm.org/index.php/Code_Formatting_Conventions